pkg/endpoints/control.go

```diff
@@ -0,0 +1,25 @@
+package endpoints
+
+import (
+	"context"
+
+	"github.com/okteto/okteto/pkg/okteto"
+)
+
+type APIControl struct {
+	OktetoClient *okteto.OktetoClient
+}
+
+func NewEndpointControl() (*APIControl, error) {
```

Comment:
Should we add the minimum interface as parameter? That way we can test it from the calling function and be able to test it from the calling function.

Reply:
52416511b7aff8495f32628b730d7ac78a10671e